### PR TITLE
fix the app and remove the warning

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang6@googlemail.com>
Fixes: #413

### CODE REVIEW

File: src/main/resources/files/BookingController.java.tpl [hunk 1]
```diff
@@ -27,7 +27,6 @@ public class BookingController {
                     content = @Content(
                             mediaType = MediaType.APPLICATION_JSON,
                             schema = @Schema(
-                                    ref = "Booking",
```
Comment (rdebusscher) [inline src/main/resources/files/BookingController.java.tpl:30]: Without the ref value, the OpenAPI document inlines the definition of Booking and no longer reference it.  This change is not wanted!

**Review by rdebusscher — CHANGES_REQUESTED** [review]

**Comment by rdebusscher** [comment]
openAPI compare

**Comment by Emily-Jiang** [comment]
@rdebusscher I did test against Open Liberty. Prior to the change, I did not see the referenced schema.
```
paths:
  /data/booking/{bookingId}:
    get:
      operationId: getBooking
      parameters:
      - name: bookingId
        in: path
        required: true
        style: simple
        schema:
          type: string
      responses:
        "200":
          description: Booking for id
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Booking'
        "404":
          description: No booking found for the id.

```
I then tested payara micro before my change (out of box) on MP 3.3 as MP 4.0  is not working (start.microprofile.io). I got the following:
```
paths:
  /data/booking/{bookingId}:
    get:
      operationId: getBooking
      parameters:
      - name: bookingId
        in: path
        required: true
        style: simple
        schema:
          type: string
      responses:
        "200":
          description: Booking for id
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Booking'
        "404":
          description: No booking found for the id.
```

**Comment by rdebusscher** [comment]
See #416

**Comment by Emily-Jiang** [comment]
I've tested against the runtimes Payara, Wildfly and they are ok. The change demonstrates the usage based on the mp openapi spec.

**Review by rdebusscher — APPROVED** [review]
LGTM

**Pull request merged**